src/app/view/createLogo.js

```diff
@@ -0,0 +1,19 @@
+import logo from '/static/assets/img/logo.png'
+
+export const createLogo = () => {
+  const img = document.createElement('img');
+  console.log(img);
+  img.classList.add('.header__game-logo__img');
+  console.log(img);
+  img.src = logo;
+  console.log(img);
+  img.addEventListener('click', () => {
+    // the page will be reload
+  });
+  const header = document.querySelector('.header__game-logo');
+  header.append(img);
+};
```

Comment:
Please remove this two lines (13 and 14). It will be good if your creatLogo function returns only one img DOM element.
This element will be injected into '.header__game-logo' div by render method in view.js

`this.render('.header__game-logo', createlogo);`